app-emulation/playonlinux/playonlinux-4.4.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+
+inherit desktop eutils python-single-r1 xdg
+
+MY_PN="PlayOnLinux"
+
+DESCRIPTION="Set of scripts to easily install and use Windows games and software"
+HOMEPAGE="https://playonlinux.com/"
+SRC_URI="https://www.playonlinux.com/script_files/${MY_PN}/${PV}/${MY_PN}_${PV}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="winbind"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND=""
+RDEPEND="${PYTHON_DEPS}
+	app-arch/cabextract
+	app-arch/p7zip
+	app-arch/unzip
+	app-crypt/gnupg
+	app-misc/jq
+	$(python_gen_cond_dep '
+		dev-python/natsort[${PYTHON_USEDEP}]
+		dev-python/wxpython:4.0[${PYTHON_USEDEP}]
+	')
+	media-gfx/icoutils
+	|| ( net-analyzer/netcat net-analyzer/openbsd-netcat )
+	net-misc/wget
+	virtual/imagemagick-tools
+	virtual/wine
+	winbind? ( net-fs/samba[winbind] )
+	x11-apps/mesa-progs
+	x11-terms/xterm
+"
+
+S="${WORKDIR}/${PN}"
+
+# TODO:
+# Having a real install script
+# It will let using LANGUAGES easily
+# How to deal with Microsoft Fonts installation asked every time ?
+# How to deal with wine version installed ? (have a better mgmt of system one)
+# Look at debian pkg: https://packages.debian.org/sid/playonlinux
+
+PATCHES=(
+	"${FILESDIR}/${PN}-4.2.4-pol-bash.patch"
+	"${FILESDIR}/${PN}-4.2.4-binary-plugin.patch"
+	"${FILESDIR}/${PN}-4.2.6-stop-update-warning.patch"
+	"${FILESDIR}/${PN}-4.4-fix-broken-version-check.patch"
+	"${FILESDIR}/${PN}-4.4-wxpython4-fix.patch"
```

Comment:
`"${FILESDIR}/${P}-..."`. you can also call them all 4.4 (as used for the first time on 4.4 in gentoo) and use ${P} instead of ${PN}-... It's up to you